webapp/src/main/java/com/box/l10n/mojito/okapi/AssetExtractionStep.java

```diff
@@ -72,13 +73,13 @@ protected Event handleTextUnit(Event event) {
         ITextUnit textUnit = event.getTextUnit();
 
         if (textUnit.isTranslatable()) {
-            String name = textUnit.getName();
+            String name = StringUtils.isEmpty(textUnit.getName()) ? textUnit.getId() : textUnit.getName();
```

Comment:
we're trying to Guava instead of Apache commons. StringUtils#isEmpty() can be replaced by Strings#isNullOrEmpty()